packages/newspringchurchapp/ios/newspringchurchapp/Info.plist

```diff
@@ -86,6 +86,10 @@
 		<string>Colfax-Regular.otf</string>
 		<string>Colfax-RegularItalic.otf</string>
 	</array>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>remote-notification</string>
+	</array>
```

Comment:
so what does this do?

Reply:
I added it because it was called out in the OneSignal docs here (https://documentation.onesignal.com/docs/react-native-sdk-setup#section-add-required-capabilities). But for context it does this: https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_updates_to_your_app_silently